<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>773</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>773</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>659</width>
    <height>773</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>621</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Press Start 2P</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(140, 140, 140);</string>
    </property>
    <property name="text">
     <string>Send it!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>632</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Press Start 2P</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>630</y>
      <width>350</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Press Start 2P</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>256</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radB_1">
       <property name="font">
        <font>
         <family>Press Start 2P</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(244, 119, 255);</string>
       </property>
       <property name="text">
        <string>Pink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radB_2">
       <property name="font">
        <font>
         <family>Press Start 2P</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 97, 255);</string>
       </property>
       <property name="text">
        <string>Purple</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radB_3">
       <property name="font">
        <font>
         <family>Press Start 2P</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(80, 159, 255);</string>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radB_4">
       <property name="font">
        <font>
         <family>Press Start 2P</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(97, 226, 255);</string>
       </property>
       <property name="text">
        <string>Ligth Blue</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
